web-console/knikubevirt/clone-vm/clone-vm.md

```diff
@@ -0,0 +1,43 @@
+# Clone VM
+
+![Offline VM in List View](img/2-0-vms.png)
+
+Cloning a VM allows the user to quickly create an identical copy of a virtual machine while powered off.
+
+![VM List View Clone option in kebab menu](img/2-1-vm-list.png)
+
+Cloning is accessed from the VM List View. All item filters should be turned on by default, ensuring that all VMs are shown to the user.
+
+The Clone option should appear in the kebab menu whether the source is online or offline.
+
+![VM modal with still running notice](img/2-2-modal.png)
+
+If the VM is powered on, a warning will appear at the top of the Clone Virtual Machine modal. Running VMs will be shut down automatically if the user proceeds with a clone and then turned back on once the clone finishes.
+
+The only required field is the name, which is pre-filled based on the source VM's name and appended with “-clone”. The user can add a description and change the new VM's Namespace (which is the same as the target by default). They can also turn off the option to start the virtual machine after the cloning process finishes.
+
+Because all required fields are filled in by default, the "Clone Virtual Machine” button is active. If the user deletes the Name field this button will become inactive.
+
+Details of the source VM's configuration are displayed for the user’s reference. These options cannot be modified here prior to cloning.
+
+The user would then click “Clone Virtual Machine”.
+
+![Clone VM power off failure](img/2-3-vm-list-error.png)
+
+If the source VM fails to shut down properly, an error toast notification would be displayed. The user would then need to power off the VM manually and re-open the Clone modal to try again.
+
+![Clone VM started](img/2-4-0-vm-list-cloning.png)
+
+If the cloning process starts successfully, the newly-cloned VM will appear in the list and a toast notification will confirm that the cloning process has begun.
```

Comment:
I mentioned this in another PR from @yfrimanm as well but in general we don't use toasts in the console to indicate a process has started when that process was just activated by the user. We used to have these type of toasts in 3.x and due to user feedback have gotten away from notifying users of their own actions. For a process that is not instantaneous, it's still valid to notify via toast when it completes. Toasts are not currently in the console so if we add them, they should use PF4.

Reply:
I agree. As long the user has some feedback that something happened, we should remove them.

- They are not ideal for displaying undergoing processes 
- Duplication to the item status on the list
- underdefined and unused in our current system.